clients/unshaded/src/test/java/tachyon/client/file/options/CompleteFileOptionsTest.java

```diff
@@ -15,34 +15,13 @@
 
 package tachyon.client.file.options;
 
-import java.util.Random;
-
 import org.junit.Assert;
 import org.junit.Test;
 
-import tachyon.conf.TachyonConf;
-
 /**
  * Tests for the {@link CompleteFileOptions} class.
  */
 public class CompleteFileOptionsTest {
-
-  /**
-   * Tests that building a {@link CompleteFileOptions} works.
-   */
-  @Test
-  public void builderTest() {
```

Comment:
Before, the `builder` may be doing something special, whereas now its a straightforward `set` (no ambiguity as to how the value will be used).